ui/tests/helpers/secret-engine/secret-engine-helpers.js

```diff
@@ -20,6 +21,32 @@ export const createSecretsEngine = (store, type, path) => {
   });
   return store.peekRecord('secret-engine', path);
 };
```

Comment:
All changes until the next comment are just moving methods around to make it easier to read